### PR TITLE
New article: Cloud Office DNS documentation update

### PR DESCRIPTION
**Pull request by WilliamLoy**
Hi content team!

Per Bryan Carter, we have updated some Cloud Office DNS settings articles. We have some formatting requests for the update that we were not able to complete on our own. 

The beginning of each articles starts with:
APPLIES TO: example
Difficulty: example
Time Needed: example
Tools Required: example

Can this please be contained in  a text box with a light blue fill?

Also for:

_Warning: content_ 

Can this please be contained in a text box with a safety orange fill?

Lastly for:

_Note: content_ 

Can this please be contained in a text box with a bright yellow fill?

We look forward to hearing back from you!

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6819a2f751/how-to/). :bow:

**Comment by fm** [comment]
Hi Content Team,

We found a misrouted link. We're fixing now and we'll commit and let you know when it's fixed.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-829db8c3e4/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9f97729ba3/how-to/)

**Comment by WilliamLoy** [comment]
Link has been fixed, you can proceed with merge. 

Thank you!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3e2d5d32b0/how-to/)

**Comment by WilliamLoy** [comment]
Hi Content Team!

I apologize for all of the updates. 

One more request, for:
Applies To: example
Difficulty: example
Time Needed: example
Tools Required: example

In addition to placing this within a text box with a light blue fill, can you make single spaces between the lines instead of the double it is now? I tried eliminating the space but this put all of the text on the same line. 

Thank you again!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cab85b0687/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e8b7c56d43/how-to/).

**Comment by WilliamLoy** [comment]
Good evening Content Team,

Admittedly this was my first pull request ever and I know it was a bit premature. 
This is now ready to merge at last

Thank you for the understanding. 

-Will

**Comment by stephamon** [comment]
Hi @WilliamLoy as for the Notes and Warnings, we do have call-out boxes in our article template. They probably aren't showing in the preview because you've got the notes in italics.

Do you have a hard deadline for when you'd like these published? If not, we'll pull this into our next sprint, which begins on Wed. May 24.

**Comment by fm** [comment]
@stephamon Bryan Carter (our Director of Support) would like this pushed by EOD today, if feasible.

File: content/rackspace-email/set-up-dns-records-for-cloud-office-email-and-skype-for-business.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+permalink: set-up-dns-records-for-cloud-office-email-and-skype-for-business/
```
Comment (stephamon) [inline content/rackspace-email/set-up-dns-records-for-cloud-office-email-and-skype-for-business.md]: Note to whoever claims this issue: this permalink needs to change to `set-up-dns-records-for-cloud-office-email/` and then have a redirect implemented in nexus-control.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-74579b9fdb/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8e53b32dff/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-45576913c1/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-adc223d626/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3d7e58512e/how-to/)

**Comment by ReneeRendon** [comment]
We will not be able to merge this PR today, but we will begin to work on it as soon as we can. 

Thanks @WilliamLoy

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9b040a7292/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-36b545aae8/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a3d7087e3e/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-31c70bdc29/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-191c49c4ce/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-91bc3c2988/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e8ba13d0ad/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-339b7fb526/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e2b190b2ac/how-to/)

**Comment by lauracly** [comment]
There isn't a need to create a box to house this content. What is valuable is the idea that you should identify the audience of the content, how difficult the task might be, time needed, and any system/software requirements (assuming this is what Tools Needed is referencing). This should really be written in sentence form, as part of the intro rather than as a new section or box. The template does already have a Prerequisite section and a description of how to write the Introduction.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e1be5f556c/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4b9b09bc12/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-cc7564981d/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-bce4bb5e28/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e391ffaed1/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-d1556dd5ca/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0a6f1e4604/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-519c96adfc/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-52b570e5d7/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-dc8a27474a/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5a406abe77/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2449f7434b/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6fde57a7bd/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b2b75bbd92/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ab37ef2bf9/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cc22188764/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3f91f14363/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-53c3cb9736/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3ac92b2f89/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b05af77fc3/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f39fa56c13/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6ec9917785/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-251af15348/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-43202b818e/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8f5a66925e/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-57a27856fa/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-31ff6bb6be/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1830b45dd0/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-72a330ad49/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f6a9d9dd70/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-031e6c4e78/how-to/). :bow:

**Comment by stephamon** [comment]
Hi @kmsholcomb I've got a big one for you! There are 9 articles in this PR that need to be edited. We'll have more articles like these coming, but they'll be 1 article/PR from now on. Let me or @WilliamLoy know if you have any questions!

**Comment by stephamon** [comment]
Redirect located here: https://github.com/rackerlabs/nexus-control/pull/729

**Comment by kmsholcomb** [comment]
Thanks, @stephamon! I'll start on them first thing tomorrow.

**Comment by WilliamLoy** [comment]
Thank you for all of the help @kmsholcomb  and @stephamon ! I am available for anything that you need.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b40317853f/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-103d67e5be/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-31f8ef6c45/how-to/)

**Comment by kmsholcomb** [comment]
Hi @stephamon. I'm posting edits for three of the articles now (Word docs and PDF copies). I'll post more edits as soon as I complete them. The edits are pretty light, but let me know if you have any questions. Thanks!

[Best practices for sending emails to many recipients_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1038634/Best.practices.for.sending.emails.to.many.recipients_editsKH.docx)
[Best practices for sending emails to many recipients_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1038633/Best.practices.for.sending.emails.to.many.recipients_editsKH.pdf)

[Best practices for sending person to person email_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1038635/Best.practices.for.sending.person.to.person.email_editsKH.docx)
[Best practices for sending person to person email_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1038637/Best.practices.for.sending.person.to.person.email_editsKH.pdf)

[Common Email Bounces_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1038636/Common.Email.Bounces_editsKH.docx)
[Common Email Bounces_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1038632/Common.Email.Bounces_editsKH.pdf)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-61e51f3a8d/how-to/)

**Comment by kmsholcomb** [comment]
Some go really quickly ... :)

[Cloud Office Blocked Attachment Types_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1038735/Cloud.Office.Blocked.Attachment.Types_editsKH.docx)
[Cloud Office Blocked Attachment Types_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1038734/Cloud.Office.Blocked.Attachment.Types_editsKH.pdf)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-b118ccd8f9/how-to/)

**Comment by kmsholcomb** [comment]
I can't even tell what changed in this file, but I had only one edit (probably reversing an edit I made a couple of years ago!). :)

[Download a Skype for Business client_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1038785/Download.a.Skype.for.Business.client_editsKH.pdf)
[Download a Skype for Business client_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1038786/Download.a.Skype.for.Business.client_editsKH.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-eaf0830423/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-69bc665ae3/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9d7eeee3b2/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2450ec0a76/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-54fbc1d349/how-to/).

**Comment by kmsholcomb** [comment]
Last of the original nine. Let me know if you need anything else! Thanks!

[Set up DNS records for Cloud Office Email_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1039687/Set.up.DNS.records.for.Cloud.Office.Email_editsKH.docx)
[Set up DNS records for Cloud Office Email_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1039689/Set.up.DNS.records.for.Cloud.Office.Email_editsKH.pdf)

[Set up DNS records for Cloud Office Skype for Business_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1039683/Set.up.DNS.records.for.Cloud.Office.Skype.for.Business_editsKH.docx)
[Set up DNS records for Cloud Office Skype for Business_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1039684/Set.up.DNS.records.for.Cloud.Office.Skype.for.Business_editsKH.pdf)

[DNS record definitions_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1039686/DNS.record.definitions_editsKH.docx)
[DNS record definitions_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1039685/DNS.record.definitions_editsKH.pdf)

[Find your DNS host_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1039690/Find.your.DNS.host_editsKH.docx)
[Find your DNS host_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1039688/Find.your.DNS.host_editsKH.pdf)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cbbcf3fb6d/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1294388994/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f72258d2be/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4e080bd64a/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6d6029be02/how-to/)

**Comment by kmsholcomb** [comment]
Hi @stephamon. Attached are suggestions for the new article. I think that using a table in the first section and a definition list in the second might improve the readability of the content. Let me know what you think. Thanks!

[Cloud Office support terminology_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1042500/Cloud.Office.support.terminology_editsKH.docx)
[Cloud Office support terminology_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1042499/Cloud.Office.support.terminology_editsKH.pdf)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-63c1473e31/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-913923dfca/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-8f2fc6b1bc/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-62b49831db/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-25de70500a/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b17e3773d8/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-edbe5397f8/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6587a310a8/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-28b057fa84/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-6918c15e45/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-32354a2151/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9cbac1953b/how-to/).

**Pull request merged**